src/pages/SupplierRequisitionsPage.js

```diff
@@ -83,28 +71,30 @@ export const SupplierRequisitionsPage = ({
     dataState,
     sortBy,
     isAscending,
-    columns,
     modalKey,
     hasSelection,
     searchTerm,
+    PageActions,
+    columns,
   } = state;
 
   // Custom hook to refresh data on this page when becoming the head of the stack again.
-  useNavigationFocusRefresh(dispatch, navigation);
+  const callback = () => dispatch(PageActions.refreshData());
+  useNavigationFocus(callback, navigation);
 
   // Custom hook to listen to sync changes - refreshing data when requisitions are synced.
-  useDatabaseListener(() => dispatch(refreshData()), 'Requisition');
+  useDatabaseListener(callback, 'Requisition');
 
   const usingPrograms = useMemo(() => getAllPrograms(Settings, UIDatabase).length > 0, []);
   const { SELECT_SUPPLIER, PROGRAM_REQUISITION } = MODAL_KEYS;
   const NEW_REQUISITON = usingPrograms ? PROGRAM_REQUISITION : SELECT_SUPPLIER;
 
   const onPressRow = rowData => () => reduxDispatch(gotoSupplierRequisition(rowData));
-  const onConfirmDelete = () => dispatch(deleteRequisitions());
-  const onCancelDelete = () => dispatch(deselectAll());
-  const onSearchFiltering = value => dispatch(filterData(value));
-  const onNewRequisition = () => dispatch(openModal(NEW_REQUISITON));
-  const onCloseModal = () => dispatch(closeModal());
+  const onConfirmDelete = () => dispatch(PageActions.deleteRequisitions());
```

Comment:
Same as before, think some of these could be memoised to save a few render cycles?